docs/setup.md

```diff
@@ -12,93 +12,138 @@
+The following instructions use [Conda](https://docs.conda.io/en/latest/) to install the tools you'll need for this tutorial.
+Conda is a package and environment management system that allows you to install Python and other software into controlled environments without disrupting other software you have installed (e.g., on your computer, your shared cluster, etc.).
 
-  * [Recommended] If you're familiar with git, clone this repository either via the web interface, a GUI such as [GitKraken](https://www.gitkraken.com/), or the command line:
+<p style="color: #055160; background-color: #cff4fc; border-color: #b6effb; padding: 1em; border-radius: .25rem;">
+If you use Microsoft Windows, [install the Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+Follow [instructions to open a new WSL window for your Linux distribution](https://docs.microsoft.com/en-us/windows/wsl/wsl-config) and then run the following commands.
+</p>
+
+### Create the Nextstrain environment
+
+[Install Miniconda with Python 3 for your operating system](https://docs.conda.io/en/latest/miniconda.html), update Conda to the latest version, and install Mamba for faster installation.
 
 ```bash
-git clone https://github.com/nextstrain/ncov.git
+conda update -n base conda
+conda install -n base -c conda-forge mamba
 ```
 
-  * [Alternative] If you're not familiar with git, you can also download a copy of these files via the buttons on the left.
+Create a Conda environment named ``nextstrain``.
+This command will install [Git](https://git-scm.com/), [Snakemake](https://snakemake.readthedocs.io/en/stable/), and [Nextstrain's full toolkit](https://docs.nextstrain.org/en/latest/install-nextstrain.html), the tools you'll need to work through this tutorial.
 
-## 2. Setup your Nextstrain environment
+```bash
+mamba create -n nextstrain -c conda-forge -c bioconda \
+  augur auspice nextstrain-cli nextalign snakemake awscli git pip
+```
 
-Create a Nextstrain conda environment with augur and auspice installed.
-If you do not have conda installed already, [see our full installation instructions for more details](https://nextstrain.org/docs/getting-started/local-installation).
-If you are running Windows, [see our documentation about setting up the Windows Subsystem for Linux (WSL)](https://nextstrain.org/docs/getting-started/windows-help).
+Confirm that the installation worked.
 
 ```bash
-curl http://data.nextstrain.org/nextstrain.yml --compressed -o nextstrain.yml
-conda env create -f nextstrain.yml
 conda activate nextstrain
-npm install --global auspice
+nextstrain check-setup --set-default
 ```
 
-## 3. Run a basic analysis with example data
-
-Run a basic workflow with example data, to confirm that your Nextstrain environment is properly configured.
-First, change into the `ncov` repository's directory.
+The final output from the last command should look like this:
 
 ```bash
-cd ncov
+Setting default environment to native.
```

Comment:
I keep getting `Setting default environment to docker.` even though I explicitly removed the Docker environment. I have to work around this by manually editing `~/.nextstrain/config`.

Reply:
Thank you for pointing this out, @fanninpm. I'll try to recreate this locally and document a better solution.

Just to make sure I understand your environment, when you say you removed the Docker environment, does that mean that the Docker daemon isn't running when you run the `nextstrain check-setup` command?

Reply:
When I ran `nextstrain check-setup --set-default`, I got the following:

```none
nextstrain-cli is up to date!

Testing your setup…

# docker is supported
✔ yes: docker is installed
✔ yes: docker run works
? unknown: containers have access to >2 GiB of memory
✔ yes: image is new enough for this CLI version

# native is supported
✔ yes: snakemake is installed
✔ yes: augur is installed
✔ yes: auspice is installed

# aws-batch is not supported
✘ no: job description "nextstrain-job" exists
✘ no: job queue "nextstrain-job-queue" exists
✘ no: S3 bucket "nextstrain-jobs" exists

All good!  Supported Nextstrain environments: docker, native
Setting default environment to docker.
```

Should the cli prefer a native environment over Docker?

Reply:
This is helpful, thank you! It looks like the Docker process is running in the background because the CLI's `docker run` test passes. The CLI was designed with Docker in mind and will prefer Docker over the native environment when possible. One way around this is to exit the Docker process and re-run the `nextstrain check-setup --set-default` command. The command should fail on the `docker run` test and default to the native environment.

Is that something you could try in your current environment? It does assume you have security privileges on your computer to turn off Docker...

Reply:
> It does assume you have security privileges on your computer to turn off Docker...

I don't even need to do that. I can just create a fake `docker` executable that will only `exit 1` and put it in the miniconda environment.

```none
nextstrain-cli is up to date!

Testing your setup…

# docker is not supported
✔ yes: docker is installed
✘ no: docker run works
? unknown: containers have access to >2 GiB of memory
✔ yes: image is new enough for this CLI version

# native is supported
✔ yes: snakemake is installed
✔ yes: augur is installed
✔ yes: auspice is installed

# aws-batch is not supported
✘ no: job description "nextstrain-job" exists
✘ no: job queue "nextstrain-job-queue" exists
✘ no: S3 bucket "nextstrain-jobs" exists

All good!  Supported Nextstrain environments: native

Setting default environment to native.
```